lib/phoenix_playground/router.ex

```diff
@@ -128,6 +128,15 @@ defmodule PhoenixPlayground.Router do
       module().handle_info(message, socket)
     end
 
+    @impl true
+    def handle_async(message, result, socket) do
+      if function_exported?(module(), :handle_async, 3) do
+        module().handle_async(message, result, socket)
+      else
+        {:noreply, socket}
+      end
```

Comment:
```suggestion
      module().handle_async(message, result, socket)
```

Reply:
I wasn't sure we needed the conditional so I went ahead and removed it. In other words, if you're using start_async/2, you should write handle_async/3. Let me know if there was a reason for the conditional though!

Reply:
@wojtekmach I basically went with copying implementation from another PR 🙈 Other than that, no real reason. Thanks for cleaning it up!